vhdbuilder/packer/list-images.ps1

```diff
@@ -5,8 +5,8 @@
     .DESCRIPTION
         Produces a JSON image BOM for a Windows VHD
 #>
-$windowsSKU = $env:WindowsSKU
-$windowsImageVersion = $env:WindowsImageVersion
+$WindowsSKU = $env:WindowsSKU
+$AKSWindowsImageVersion = $env:AKSWindowsImageVersion
```

Comment:
it seems like that there is no reason to change the first letter.
$windowsSKU
$aksWindowsImageVersion